Users/Tutorial.rst

```diff
@@ -54,14 +54,15 @@ There are two kinds of bears:
 - LocalBears, which only perform analysis on each file itself
 - GlobalBears, which are project wide, like the GitCommitBear
 
-Let's find out what bears are available for C:
+Let's find out what bears are available for C and Python:
 
 ::
 
-    coala --show-bears --filter-by-language C
+    coala --show-bears --filter-by-language C Python
 
-coala now lists all bears that can operate on C code. You can specify the
-kind of analysis with the ``--bears`` argument:
+coala now lists all bears that can operate on C or Python code.
+Note that here you can give space separated language names as muliple
+arguments. You can specify the kind of analysis with the ``--bears`` argument:
```

Comment:
can we have two newlines before the `You can spec...`? I'd also rephrase that: `You can specify the bears you want coala to run using the --bears argument.` (plus the backticks around --bears)